ISS/models/core_models.py

```diff
@@ -221,13 +222,27 @@ def can_auto_anonymize(self):
     @transaction.atomic
     def merge_into(self, other):
         """
-        Reassigns all the records that tie to this user to another one and
+        Reassigns all (<-- lies) the records that tie to this user to another one and
         disables this user afterwards.
         """
         
         Thread.objects.filter(author=self).update(author=other)
+
+        Post.objects.filter(author=self).update(posted_from='127.0.0.1')
         Post.objects.filter(author=self).update(author=other)
 
+        PostSnapshot.objects.filter(obsolesced_by=self).update(obsolescing_ip='127.0.0.1')
+        PostSnapshot.objects.filter(obsolesced_by=self).update(obsolesced_by=other)
+
+        for post in Post.objects.all():
+            post.content = post.content.replace("author=\"" + self.username + "\"", "author=\"" + str(other) + "\"")
+            post.save()
```

Comment:
I know people don't like having their usernames hang out in quotes, but we can't do this for two reasons:

1. On a policy/moral level this edits posts belonging to people other than the person anonymizing themselves. Editing other peoples posts is a right reserved for the tyrant (me)
2. Iterating over every post in the DB, doing a string substitution, and writing back to the DB is going to clobber the DB. On NiS prod the select query probably won't even finish before we run out of memory.